site/package.json

```diff
@@ -26,5 +26,10 @@
     "shortcut-buttons-flatpickr": "0.3.0",
     "twig": "1.15.1"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "eslint": "^7.1.0"
+  },
+  "scripts": {
+    "lint": "eslint public/js/*.js public/js/pdf/*.js"
```

Comment:
Can we also have a script for auto fixing using eslint
and is there a way to pass in a specific file as an argument instead of hitting every file?
something like:
```
eslint --fix --fix-type problem,layout <file>
```

Reply:
I was going to add a `"lint:fix": "eslint public/js/*.js public/js/pdf/*.js"` run-script. You can also activate eslint directly by doing `node_modules/.bin/eslint`. Yes, I will add documentation once we're agreed on the rules.

Reply:
added.